plugin/controller/src/main/java/org/team199/deepbluesim/mediators/CANMotorMediator.java

```diff
@@ -39,19 +41,19 @@ public class SimDeviceMotorMediator implements Runnable {
      * @param gearing the gear ratio to use
      * @param callbackStore a collection to store callbacks in
      */
-    public SimDeviceMotorMediator(Motor motor, SimDeviceSim simDevice, DCMotor motorConstants, double gearing, boolean inverted, Collection<ScopedObject<?>> callbackStore) {
+    public CANMotorMediator(Motor motor, CANMotorSim simDevice, DCMotor motorConstants, double gearing, boolean inverted, Collection<ScopedObject<?>> callbackStore) {
         this.motor = motor;
         motorDevice = simDevice;
         this.motorConstants = motorConstants;
         this.gearing = gearing;
         this.inverted = inverted;
 
-        if(motor.getName().startsWith("DBSim_Motor_Spark Max")) {
+        if(motor.getName().startsWith("DBSim_Motor_Spark ")) {
```

Comment:
To be consistent with the check in `SimRegisterer.java`
```suggestion
        if(motor.getName().startsWith("DBSim_Motor_Spark")) {
```